scripts/doctor

```diff
@@ -29,18 +29,18 @@ warn() {
 found=0
 count_in_path() {
     echo "Check for ${1} in path..."
-    found=$(which -a "${1}" | uniq | wc -l)
+    found=$(which -a ${1} | uniq | wc -l)
 }
 
 
-count_in_path "rbenv"
+count_in_path "rbenv asdf"
 if [ $found -eq 0 ]; then
     exitcode=1
-    warn "yel" "   rbenv not found. See https://github.com/rbenv/rbenv for information on installing it."
+    warn "yel" "   rbenv or asdf not found. See https://github.com/rbenv/rbenv for information on installing it."
```

Comment:
Might as well add both here for clarity. Maybe `Could not find rbenv or asdf. See https://github.com/rbenv/rbenv for information on installing rbenv (recommended), or https://asdf-vm.com/guide/getting-started.html for information on installing asdf.`